src/api/bundler-namespace.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  UserOperation,
+  UserOperationEstiamateGasResponse,
+  UserOperationReceipt,
+  UserOperationRequest
+} from '../types/types';
+import { AlchemyConfig } from './alchemy-config';
+
+/**
+ * The Bundler namespace contains methods used for interacting performing user operations and
+ * checking on the state of submitted operations.
+ *
+ * Do not call this constructor directly. Instead, instantiate an Alchemy object
+ * with `const alchemy = new Alchemy(config)` and then access the transact
+ * namespace via `alchemy.paymaster`.
+ */
+export class BundlerNamespace {
+  /** @internal */
+  constructor(private readonly config: AlchemyConfig) {}
+
+  /**
+   * Sends a User Operation
+   *
+   * Sends a user operation to the provided bundler client evm network, and
+   * returns a user operation hash on successful submission.
+   *
+   * @param userOperation - The user operation.
+   * @public
```

Comment:
instead of using `@public` here, use `@beta` here and throughout for now, which lets us do breaking changes on the method later.